components/spotify/actions/add-items-to-playlist/add-items-to-playlist.mjs

```diff
@@ -0,0 +1,53 @@
+import lodash from "lodash";
+import spotify from "../../spotify.app.mjs";
+
+export default {
+  name: "Add items to a playlist",
```

Comment:
I normally try to match the case of the operation in the API docs. [In Spotify's case](https://developer.spotify.com/documentation/web-api/reference/#category-playlists) this is: "Add Items to a Playlist"